vllm_ascend/ops/fused_moe.py

```diff
@@ -590,27 +620,55 @@ def __init__(self,
             self.local_num_experts, self.expert_map = determine_expert_map(
                 self.ep_size,
                 get_ep_group().rank_in_group, self.global_num_experts)
-            self.tp_rank = get_etp_group().rank_in_group
-            self.ep_rank = get_ep_group().rank_in_group
+            if vllm_version_is("0.8.5") or vllm_version_is("0.8.5.post1"):
+                self.tp_rank = get_etp_group().rank_in_group
+                self.ep_rank = get_ep_group().rank_in_group
+            else:
+                self.moe_parallel_config.tp_rank = get_etp_group(
+                ).rank_in_group
+                self.moe_parallel_config.ep_rank = get_ep_group().rank_in_group
+
         else:
             # Adjust TP size for DP attention
             # haven't test its functionality yet, may remove in the future
-            self.tp_rank = self.tp_size * self.dp_rank
-            self.ep_rank = 0
-            self.tp_size = self.tp_size * self.dp_size
-            self.ep_size = 1
-            self.local_num_experts = self.global_num_experts
-            self.expert_map = None
-
+            if vllm_version_is("0.8.5") or vllm_version_is("0.8.5.post1"):
```

Comment:
I think this part of the code may not be needed, refer to the modification of this part in [PR 863](https://github.com/vllm-project/vllm-ascend/pull/863)

However, the most urgent thing at present is to fix CI, which can be considered later

Reply:
yes  let' make ci happy first then solve the bug later